### ISSUE TITLE
Not easy to distinguish between two alerts showing on the same page

### ISSUE THREAD

**Issue by keelanfh** [localgovdrupal/localgov_alert_banner#237]
Picked up from some accessibility work we had done:

> if there is more than one alert then it is not easy to distinguish between either ‘Hide alert’ button as their text content is identical. Recommend adding aria-describedby to each hide button and reference an ID set on the component heading

**Comment by andybroomfield** [comment]
Hmm, think it should be there? (as an aria-label rather than aria-described-by).
`<button class="localgov-alert-banner__close js-localgov-alert-banner__close" aria-label="{{ 'Hide alert'|t ~ ': ' ~ content.title|render|striptags|trim }}">{{ 'Hide'|t }}</button>`
https://github.com/localgovdrupal/localgov_alert_banner/blob/d984820266ddaab7a7bf3cbb7f5a7a2936422091/templates/localgov-alert-banner.html.twig#L60

Seems to be present when I check.
<img width="1170" alt="Screenshot 2023-02-15 at 1 48 44 pm" src="https://user-images.githubusercontent.com/1467480/219045256-485d708d-453c-4c10-a962-9994658fa308.png">

No objection to changing, as long as its possible to work out the ID of the title (which might be hidden).

**Comment by danchamp** [comment]
The heading is optional, so it's not a reliable descriptor for the button since it may not be present in markup (not just hidden).

**Comment by keelanfh** [comment]
Ah, I think this is also one that's being overridden in localgov_base:

https://github.com/localgovdrupal/localgov_base/blob/32261e5237ceb20012de41e1f4fd103efa1aace1/templates/block/localgov-alert-banner.html.twig#L67

Regardless, can still look at selecting the most reliable descriptor.

**Comment by danchamp** [comment]
Good spot @keelanfh.

**Comment by danchamp** [comment]
Dupe of [localgov_base #146](https://github.com/localgovdrupal/localgov_base/issues/146)?

**Comment by keelanfh** [comment]
No, I think they're two different issues (though we did spot them together).
